### ISSUE TITLE
Validation methods break when setting `null` values

### ISSUE THREAD

**Issue by broofa** [allmas-tn/sequelize-transforms#1]
I have a Sequelize model, "User", with the following column definition:
```
sms: {
  type: DataTypes.STRING,
  allowNull: true,
  trim: true
}
```

When I do this:

```
user = User.build();
user.sms = null;
```

I get this:

```
tend:boss TypeError: Cannot read property 'toString' of null
tend:boss   at Object.trim (node_modules/sequelize-transforms/index.js:8:36)
tend:boss   at Instance.<anonymous> (node_modules/sequelize-transforms/index.js:32:36)
tend:boss   at node_modules/sequelize-transforms/index.js:46:22
```

**Comment by broofa** [comment]
Whups, just saw that this is fixed on v1.0.1.  Leaving open since that version hasn't been `npm publish`'ed.  Perhaps you could do that after merging the PR I just posted?

**Comment by Erid** [comment]
Everyone could publish their own versions while we wait... But just in case we published `@gopato/sequelize-transforms` (we based off @broofa's PR)

**Comment by hbj** [comment]
Sorry @broofa for being so silent, I was not receiving any notifications. I'll check this asap.

**Comment by hbj** [comment]
Fixed in a118b9c.
